src/sync/installation.ts

```diff
@@ -110,7 +110,19 @@ export const updateJobStatus = async (
 	const status = isComplete ? "complete" : "pending";
 
 	logger.info({ status }, "Updating job status");
-	await updateRepo(subscription, repositoryId, { [getStatusKey(task)]: status });
+
+	const values: { [x: string]: string | Date} = { [getStatusKey(task)]: status };
+
+	if (isComplete && task === "commit" && data.commitsFromDate) {
```

Comment:
/nit
Just to clarify, once we implement `commitFrom` for all tasks, this `if ()` branch will be generic, right?

Reply:
Yep, then we can remove task condition in above expression